conf/charts/redis-janitor/values.yaml

```diff
@@ -24,6 +24,7 @@ rbac:
   pspEnabled: false
 
 env:
+  QUEUES: "predict,track"
```

Comment:
Do we need to have a "train" queue listed here, since the autoscaler also has a "train" queue?

Reply:
The "train" flow has been disabled for now, but we could potentially have it check the key anyway (there just will be no values).